oidc/identity.go

```diff
@@ -8,10 +8,11 @@ import (
 )
 
 type Identity struct {
-	ID        string
-	Name      string
-	Email     string
-	ExpiresAt time.Time
+	ID              string
+	Name            string
+	Email           string
+	ExpiresAt       time.Time
+	AdditonalClaims jose.Claims
 }
 
 func IdentityFromClaims(claims jose.Claims) (*Identity, error) {
```

Comment:
I intend to add that functionality.  It depends on some of the functionality needed for issue #14.  I'm preparing another PR that would address this.